check-node-version.js

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+
+fs.readFile(path.join(__dirname, '.nvmrc'), 'utf-8', function (err, data) {
```

Comment:
What is the benefit of using the async version of `readFile` here? This would be easier to maintain and faster if we used `fs.readFileSync()`.